examples/connectivity/plot_adhd_tangent.py

```diff
@@ -0,0 +1,107 @@
+"""
+Comparing different connectivity measures
+=========================================
+
+This example compares different measures of functional connectivity between
+regions of interest.
+
+"""
+
+# Fetch dataset
+import nilearn.datasets
+atlas = nilearn.datasets.fetch_atlas_msdl()
+dataset = nilearn.datasets.fetch_adhd()
+
+# Extract and preprocess regions time series
+import nilearn.input_data
+import joblib
+mem = joblib.Memory('/home/sb238920/CODE/Parietal/nilearn/nilearn_cache/adhd')
+masker = nilearn.input_data.NiftiMapsMasker(
+    atlas.maps, resampling_target="maps", detrend=True,
+    low_pass=None, high_pass=None, t_r=2.5, standardize=False,
```

Comment:
Yes here I am !
I removed filtering and confounds regression on purpose: tangent behaves better with noisy data. Its score falls down when filtering.
I don't standardize because 
- I want to show the difference between covariance and correlation measure
- standardizing ruins tangent score
- the tangent mean has not unit diagonal even if individual time series are standardized